src/ajax.js

```diff
@@ -5,6 +5,9 @@ var utils = require('./utils');
 const XHR_DONE = 4;
 let _timeout = 3000;
 
+export const GET = 'GET';
+export const POST = 'POST';
```

Comment:
Ironically... I actually added these because I couldn't remember where this ajax function was expecting `GET` or `get`. Figured this would save me from having to look it up the next time.

I must admit that I don't care much either way, though. I wouldn't mind voting on it (thumbs up for `const`, thumbs down for raw strings)

Reply:
If this were our API, then I'd probably be more open to the idea of adding these constants.  But this is just a layer on top of an existing API that's already pretty standardized around using 'GET', 'POST', etc strings: https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/open

Reply:
aha... ok, that makes plenty of sense then. My javascript inexperience is showing itself... didn't know those were so well established.

I'll take em out.